src/en/actions.md

```diff
@@ -109,6 +109,29 @@ with the following command:
 juju run-action git/0 remove-repo --params=params.yaml sure=yes
 ```
 
+If you have an action that requires multiple lines, use YAML quoting to make
+sure the whitespace is not collapsed into one line, like in this example where
+`foo` is an action and the parameter `bar` is defined in the example 
+`actions.yaml`, shown just after the example:
+
```

Comment:
I think the above sentence could do with a slight rewording to explain that the YAML quoting is why there's both " and ' below, and to perhaps remove the multiple uses of the word 'example' (which is confusing my tiny brain).